README.md

```diff
@@ -1,34 +1,83 @@
+  <a href="https://github.com/Dessia-tech/volmdlr/issues"><img src="https://img.shields.io/github/issues/Dessia-tech/volmdlr.svg"></a>
+</div>
+
+<div align="center">
+  <a href="#description"><b>Description</b></a> |
+  <a href="#features"><b>Features</b></a> |
+  <a href="#requirements"><b>Requirements</b></a> |
+  <a href="#installation"><b>Installation</b></a> |
+  <a href="https://github.com/Dessia-tech/volmdlr/tree/master/scripts"><b>Usage</b></a> |
+  <a href="https://documentation.dessia.tech/volmdlr/"><b>Documentation</b></a> |
+  <a href="#licence"><b>Licence</b></a> |
+  <a href="#contributors"><b>Contributors</b></a> |
+</div>
+
+## Description
+
+Volmdlr is a python volume modeler used as a CAD platform.
+It is simple to understand and operate.
+With it, you can easily create 3D models.
+Check the exemples to see what you can do with this library.
+
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/casing.jpg" width="42%" />
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/casing_contours.png" width="57%" /><br/>
+<i>A casing is defined by a 2D contour formed with the primitive RoundedLineSegment2D. This contour is offset by the casing width.</i><br/>
+
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/sweep1.jpg" width="47%" />
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/sweepMPLPlot.jpg" width="52%" /><br/>
+<i>A Sweep is pipes, created with Circle2D/Arc2D which is contained in a Contour2D. You have to create the neutral fiber, i.e., the pipe’s road, with the primitive RoundedLineSegment3D.</i><br/>
+
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/polygon.jpg" width="47%" /><br/>
+<i>A polygon is defined out of points. Random points are sampled and the tested whether they are inside or outside the polygon. They are plotted with the Matplotlib binding MPLPlot with custom styles:
+- red if they are outside,
+- blue if they are inside
+</i>
+
+<img src="https://raw.githubusercontent.com/Dessia-tech/volmdlr/master/doc/source/images/bspline_surface_split.jpg" width="47%" /><br/>
+<i>A 3D B-spline surface splitted by a 3D B-spline curve.</i>
 
 ## Features
 
-* Generate 3D geometries from python
-* Primitives provides computational tasks (distances, belonging, union, intersections...)
-* STEP/STL import and exports
+- [x] Generate 2D and 3D geometries from python
+- [x] Handles complexe geometries : B-spline curves and surfaces
+- [x] Primitives provide computational tasks : distances, belonging, union, intersections, etc.
+- [x] STEP/STL imports and exports
+- [x] Geometries display in your web browser with [babylon.js](https://www.babylonjs.com/)
 
-## Installing
+## Requirements
```

Comment:
split user install/ dev install: require C compiler only for dev (pip provides mac & windows whl)